CHANGELOG.md

```diff
@@ -254,11 +258,14 @@ Backwards compatible changes
 
 * Added support for GHC 8.2.2.
 
-* New modules `Data.Table`, `Data.Table.Base`,
-  `Data.Table.Relation.Equality` and `Data.Table.Properties`. A `Table` is a
-  fixed-length collection of objects similar to a `Vec` from `Data.Vec`, but
-  implemented as a function `Fin n → A`. This prioritises ease of lookup as opposed
-  to `Vec` which prioritises the ease of adding and removing elements.
+* New module `Data.Word` for new builtin type `Agda.Builtin.Word.Word64`.
```

Comment:
You probably didn't mean to commit this

Reply:
Thanks, nice spot! Didn't branch off the right place, and clearly mishandled the merge.